### ISSUE TITLE
Add healthcheck to replica sync-agent sidekick

### ISSUE THREAD

**Issue by cjellick** [rancher/longhorn#34]
Blocked by https://github.com/rancher/rancher/issues/4688

**Comment by cjellick** [comment]
Not a very high priority since if the process crashes, our service reconcile will catch it and restart the controller.

**Comment by cjellick** [comment]
I'm moving this one out of beta. For two reasons, 1) the blocking bug in rancher is not yet addressed 2) missing the healthcheck has no significant impact on functionality. Its just a nice-to-have.

Note: perhaps a work-around would be to incorporate a check on the sync-agent's health into the existing replica healthcheck.

**Comment by cjellick** [comment]
I might close this. I'm not sure if its something we'll ever add. Adding unrequired healthchecks is just unnecessary overhead.
